Rakefile

```diff
@@ -121,12 +117,11 @@ def beaker_test(mode = :base, options = {})
   overriding_options = ENV['OPTIONS'].to_s
 
   args = [options_opt, hosts_opt(preserved_hosts_mode), tests_opt,
-          *overriding_options.split(' ')].compact
+          *overriding_options.split(' '),].compact
```

Comment:
This looks incorrect. Not sure how to teach RuboCop that it shouldn't be there.

Reply:
this comes from:
```
Rakefile:120:11: C: [Corrected] Style/TrailingCommaInArrayLiteral: Put a comma after the last item of a multiline array. (https://rubystyle.guide#no-trailing-array-commas)
          *overriding_options.split(' ')].compact
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

we have this in the rubocop.yml:
```
Style/TrailingCommaInArrayLiteral:
  Enabled: True
  EnforcedStyleForMultiline: consistent_comma
```